reviewing/license-scanner/src/getPkgInfo.ts

```diff
@@ -126,10 +140,18 @@ async function parseLicense (
       } else {
         licenseContents = await readLicenseFileFromCafs(opts.cafsDir, licensePackageFileInfo as PackageFileInfo)
       }
+      const licenseContent = licenseContents?.toString('utf-8')
+      let name = 'Unknown'
+      if (licenseContent) {
+        const match = licenseContent.match(new RegExp(`\\b(${LICENSE_NAMES.join('|')})\\b`, 'igm'))
```

Comment:
The license-finder project has a small list of headers to license definitions that would be useful:
https://github.com/pivotal/LicenseFinder/blob/master/lib/license_finder/license/definitions.rb

Reply:
Should we list both short names and other names?
https://github.com/pivotal/LicenseFinder/blob/c5bfc394afdecd48bf9a1c1b045e605ec0938937/lib/license_finder/license/definitions.rb#L58-L73

Reply:
I think I would go for the value of the `pretty_name` or `spdx_id` fields

Reply:
Perhaps the `short_name` field should be included as well, it seems to be more common.